packages/Sortable/src/Sortable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuidv4 from "uuid/v4";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import SortableContext from "./SortableContext";
+import Item from "./components/Item/Item";
+import sortableStyles from "./Sortable.styles";
+
+const propTypes = {
+  children: PropTypes.node,
+  hasIndexes: PropTypes.bool,
+  onChange: PropTypes.func.isRequired,
+  onRemove: PropTypes.func,
+};
+
+const defaultProps = {
+  children: null,
+  hasIndexes: true,
+  onRemove: null,
+};
+
+function augmentChildren(children) {
+  return React.Children.map(children, (child, index) => {
+    return child.type
+      ? React.cloneElement(child, { ...child.props, "data-drag-id": index })
+      : React.createElement("span", { ...child.props, "data-drag-id": index }, child);
+  });
+}
+
+const Sortable = ({ children, onChange, hasIndexes, onRemove }) => {
+  const augmentedChildren = augmentChildren(children);
+
+  const handleDragEnd = result => {
+    const { source, destination } = result;
+    onChange({
+      source: source.index,
+      destination: destination ? destination.index : null,
+    });
+  };
+
+  return (
+    <SortableContext.Provider value={{ hasIndexes, onRemove }}>
```

Comment:
const value = {hasIndexes, onRemove};
<SortableContext.Provider value={value}>

https://reactjs.org/docs/context.html#caveats
> assigning a value directly might create an undesirable rerender.

Reply:
Heh, just extracting the object literal into a variable changes nothing, it's still recreated on every render call. You would want to `useMemo` this object (assuming we still want to use context here)